go.mod

```diff
@@ -5,6 +5,7 @@ require (
 	github.com/ajstarks/svgo v0.0.0-20181006003313-6ce6a3bcf6cd // indirect
 	github.com/dgrijalva/jwt-go v0.0.0-20160705203006-01aeca54ebda
 	github.com/gorilla/handlers v0.0.0-20170224193955-13d73096a474
+	github.com/juju/charmstore v5.5.1+incompatible // indirect
```

Comment:
This shouldn't be being imported, I guess there is a bad import somewhere.

Reply:
"go mod tidy" fixed the issue